lib/src/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -33,17 +34,19 @@ export class BreadcrumbsComponent implements OnInit, OnDestroy {
         this.breadcrumbs = this.getBreadcrumbs(root);
 
         // subscribe to the NavigationEnd event
-        this.routerSubscription = this.router.events.filter(event => event instanceof NavigationEnd).subscribe(event => {
-            // set breadcrumbs
-            root = this.activatedRoute.root;
-            this.breadcrumbs = this.getBreadcrumbs(root);
-            if ( this.breadcrumbs.length > 1 ) {
-                this.backURL = this.breadcrumbs[ this.breadcrumbs.length - 2 ].url;
-                this.backShow = 'inline';
-                this.locationLabel = '';
-            } else {
-                this.backShow = 'none';
-                this.locationLabel = this.breadcrumbs[ this.breadcrumbs.length - 1 ].label;
+        this.routerSubscription = this.router.events.subscribe((event: Event) => {
+            if (event instanceof NavigationEnd) {
```

Comment:
Just curious the advantage of putting an `if` statement around the entire function body as opposed to using `filter`.

Reply:
This was being a pain. WIth rollup I wasn't able to get `filter` to be recognized and the build would fail. This is just the hacky way I could get the build to succeed.

You may ask why was it passing before? Because breadcrumbs weren't included in the `public_api.ts` export and rollup wouldn't transpile them